src/browser/auth/SignInPage.react.js

```diff
@@ -22,11 +23,12 @@ export default class SignInPage extends Component {
           }
         </FormattedMessage>
         <p>
-          Este uses <a
+          Este uses <Link
+            external
             href="https://firebase.google.com/docs/auth/"
             rel="noopener noreferrer"
```

Comment:
I think rel="noopener noreferrer" should be default with link to the explanation https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/